llama_index/indices/base.py

```diff
@@ -189,23 +189,61 @@ def insert(self, document: Document, **insert_kwargs: Any) -> None:
 
     @abstractmethod
     def _delete(self, doc_id: str, **delete_kwargs: Any) -> None:
```

Comment:
Should we just rename this to `delete_node` to be explicit as well?

Reply:
`_delete_node(doc_id)` 🥲

Reply:
yea can do that too haha